contracts/lpp/src/contract/lender.rs

```diff
@@ -251,7 +269,69 @@ mod test {
         let balance_nlpn = query_balance(deps.as_ref().storage, Addr::unchecked("lender2"))
             .unwrap()
             .balance;
-        assert_eq!(balance_nlpn.u128(), zero);
+        assert_eq!(balance_nlpn.u128(), 0);
+    }
+
+    #[test]
+    fn test_min_utilization() {
```

Comment:
please provide short and single-purpose tests

Reply:
Ok. I'll split it.